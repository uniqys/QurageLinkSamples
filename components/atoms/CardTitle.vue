<template>
  <p class="card-header-title">
    <slot />
  </p>
</template>

<script lang="ts">
import { Component, Vue } from 'nuxt-property-decorator'

@Component
export default class CardTitle extends Vue {}
</script>
