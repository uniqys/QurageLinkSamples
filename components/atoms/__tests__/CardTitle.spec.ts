import { shallowMount } from '@vue/test-utils'
import CardTitle from '~/components/atoms/CardTitle.vue'

describe('CardTitle.vue', () => {
  it('should to match snapshot', () => {
    const title = 'title text'
    const wrapper = shallowMount(CardTitle, {
      slots: {
        default: title
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
