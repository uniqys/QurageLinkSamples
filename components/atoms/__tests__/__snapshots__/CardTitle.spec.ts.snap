// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTitle.vue should to match snapshot 1`] = `
<p
  class="card-header-title"
>
  title text
</p>
`;
